src/components/CarouselSlider/index.tsx

```diff
@@ -29,15 +29,15 @@ const carouselStyles = cva(
 interface SlideItem {
```

Comment:
can we create all these interfaces in another file.
Only props interface should be in the file.